.circleci/config.yml

```diff
@@ -19,8 +19,10 @@ jobs:
       - run:
           name: Install Packages
           command: |
+              echo 'deb http://ftp.debian.org/debian stretch-backports main' >> /etc/apt/sources.list
               apt update
-              apt install -y libltdl-dev netcat
+              apt install -y libltdl-dev netcat clang
+              apt-get -y -t stretch-backports install git
```

Comment:
interesting that git isn't in the mainline repo

Reply:
yeah 2.18 was just released in Jun. I think vgo is using Transfer protocol v2 for partial clones.